src/main/java/com/softserve/demo/exceptions/FailedToVerifyException.java

```diff
@@ -0,0 +1,13 @@
+package com.softserve.demo.exceptions;
+
+/**
+ * This class represents Exception when some error has occurred while verification
+ * or it cannot be performed for some reasons.
+ *
+ * @author Dmytro Narepekha
+ */
+public class FailedToVerifyException extends RuntimeException {
```

Comment:
VerificationFailedException sounds better

Reply:
Thanks! Sounds great! I will change it